pkg/controller/command/command_config_maps.go

```diff
@@ -1,15 +1,19 @@
 package command
 
 import (
+	"fmt"
 	"strconv"
 
+	"github.com/google/uuid"
 	corev1 "k8s.io/api/core/v1"
 
 	contrail "github.com/Juniper/contrail-operator/pkg/apis/contrail/v1alpha1"
 	"github.com/Juniper/contrail-operator/pkg/certificates"
 	"github.com/Juniper/contrail-operator/pkg/k8s"
 )
 
+var NameSpaceCommand = uuid.Must(uuid.Parse("362ad2d9-9460-4f65-a702-2cd99fbb0f64"))
```

Comment:
i don't understand the name of this variable, what did you mean?

Reply:
Added a comment